presto-main/pom.xml

```diff
@@ -58,7 +58,7 @@
 
         <dependency>
             <groupId>com.google.code.findbugs</groupId>
-            <artifactId>annotations</artifactId>
+            <artifactId>jsr305</artifactId>
```

Comment:
`<optional>true</optional>`?

Reply:
It's used by `FunctionsParserHelper#containsLegacyNullable`